src/include/simeng/kernel/LinuxProcess.hh

```diff
@@ -90,6 +114,15 @@ class LinuxProcess {
   /** The entry point of the process. */
   uint64_t entryPoint_ = 0;
 
+  /** Program header table address */
+  uint64_t phdrTableAddress_;
+
+  /** Number of program headers */
+  uint64_t phnum_;
+
+  /** Size of program header entry */
+  uint64_t phent_;
```

Comment:
Given these are not used outside setting the Aux-Vector, why do they need to be member variables?

Reply:
I had the same initial thought. The elf object is not available outside of the LinuxProcess constructor and so is not available within createStack. Storing as a private member was the simplest solution here though they could be passed as arguments to the function. This does bring up the question of what should happen when using the other LinuxProcess constructor which does not use the elf object